.circleci/scripts/validate-locales-only.ts

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs').promises;
+const path = require('path');
+
+const BASE_PATH = path.resolve(__dirname, '..', '..');
+const CHANGED_FILES_PATH = path.join(
+  BASE_PATH,
+  'changed-files',
+  'changed-files.txt',
+);
+
+/**
+ * Reads the list of changed files from the git diff file.
+ *
+ * @returns An array of changed file paths.
+ */
+async function readChangedFiles(): Promise<string[]> {
+  try {
+    const data = await fs.readFile(CHANGED_FILES_PATH, 'utf8');
+    const changedFiles = data.split('\n');
+    return changedFiles;
+  } catch (error) {
+    console.error('Error reading from file:', error);
+    return [];
```

Comment:
Do we want this check to pass if this fails? That appears to be the current behavior. Perhaps we should remove this `try..catch` block instead, so that this fails if we're unable to properly validate changes.

Reply:
Hey, this is a really good point. I added something to help in 7d678744b5a0fd5d1d52eb6e929dda9535a0abb1

Reply:
Really good catch! I ended up importing that `readChangeFiles` method in [8e9400f](https://github.com/MetaMask/metamask-extension/pull/25809/commits/8e9400fcc4f55edcc1615d9f5ee0ee8ba86b28d3), so I'm just making sure to address your comment in [9fb0081](https://github.com/MetaMask/metamask-extension/pull/25809/commits/9fb00818fd5ae6816b1e3f4dd0a05b5b4aede0ec)